control-plane/helper/webhook-configuration/webhook_configuration.go

```diff
@@ -25,11 +25,12 @@ func UpdateWithCABundle(ctx context.Context, clientset kubernetes.Interface, web
 	}
 
 	mutatingWebhookCfg, err := clientset.AdmissionregistrationV1().MutatingWebhookConfigurations().Get(ctx, webhookConfigName, metav1.GetOptions{})
-	if err == nil {
-		return updateMutatingWebhooksWithCABundle(ctx, clientset, mutatingWebhookCfg, caCert)
+	if err != nil && !k8serrors.IsNotFound(err) {
```

Comment:
so the TLDR is that rather than just return out from the mutating webhook update we should update both the validating webhooks and mutating webhooks, I was initially under the impression that each was named after their path but it looks like there's just a single `consul-connect-injector` config name that all the validating and mutating webhooks are just paths on